packages/types/index.d.ts

```diff
@@ -12,10 +12,14 @@ export interface Stream<A> {
   run (sink: Sink<A>, scheduler: Scheduler): Disposable;
 }
 
+export type sinkEventCallback<A> = (time?: Time, value?: A) => mixed
```

Comment:
TypeScript does not have a `mixed` type as Flow does. I believe `any` is the closest match that we would have without maybe defining our own "mixed" union type.